packages/sdk/src/utils/merkle-utils.ts

```diff
@@ -37,9 +37,11 @@ export const makeMerkleTreeProof = (
     return fromHexString(ethers.utils.keccak256(el))
   })
 
-  const proof = tree.getProof(bufLeaves[index], index).map((element: any) => {
-    return toHexString(element.data)
-  })
+  const proof = tree
+    .getProof(bufLeaves[index] as any, index)
+    .map((element: any) => {
+      return toHexString(element.data)
+    })
```

Comment:
This change (adding an any) should be unnecessary but I'll test that in a follow up pr so it doesn't block this one